README.md

```diff
@@ -1,11 +1,13 @@
 [![Travis Build](https://travis-ci.org/aeternity/elixir-node.svg?branch=master)](https://travis-ci.org/aeternity/elixir-node)
 
+# CURRENTLY *NOT COMPATIBLE* TO THE AETERNITY NETWORK 
+This implementation aims to be a full node using the aeternity specification, in the current state this is not ready yet and should never be used to join the aeternity network.
```

Comment:
I think "...aims to be a full node that complies with the aeternity specification..." would be better